### PR TITLE
adds conversation handler and operations integration backup tests

### PR DESCRIPTION
**Pull request by ryanfkeepers**
#### Does this PR need a docs update or release note?

- [x] :no_entry: No

#### Type of change

- [x] :sunflower: Feature

#### Issue(s)

* #4536

#### Test Plan

- [x] :zap: Unit test
- [x] :green_heart: E2E

### CODE REVIEW

**Comment by meain** [comment]
Woops, looks like this now has a conflict. Will need a rebase before review @ryanfkeepers

**Review by meain — APPROVED** [review]

File: src/internal/operations/test/group_test.go [hunk 1]
```diff
@@ -79,6 +80,124 @@ func (suite *GroupsBackupIntgSuite) TestBackup_Run_incrementalGroups() {
...
+	// The number of items backed up in the forced full backup should be roughly
+	// the same as the number of items in the original backup.
+	assert.Equal(
+		t,
+		bo.Results.Counts[string(count.PersistedNonCachedFiles)],
+		forcedFull.Results.Counts[string(count.PersistedNonCachedFiles)],
+		"items written")
+}
+
+func (suite *GroupsBackupIntgSuite) TestBackup_Run_groupsVersion9AssistBases() {
```
Comment (pandeyabs) [inline src/internal/operations/test/group_test.go:192]: This test already exists in `GroupsBackupNightlyIntgSuite`. Any reason for duplicating it to `GroupsBackupIntgSuite`?

**Review by pandeyabs — APPROVED** [review]
LGTM, other than a test duplication question.

File: src/internal/m365/collection/groups/conversation_handler.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	return bh.ac.GetConversationThreadPostIDs(
+		ctx,
+		bh.protectedResource,
+		conversationID,
+		prevDelta,
+		cc)
+}
+
+func (bh conversationsBackupHandler) includeContainer(
+	ctx context.Context,
```
[the remark this answers is not in the document]
Comment (ryanfkeepers) [reply src/internal/m365/collection/groups/conversation_handler.go:58]: removed

File: src/internal/m365/collection/groups/conversation_handler.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+	return folders.
+		ToDataLayerPath(
+			tenantID,
+			bh.protectedResource,
+			path.GroupsService,
+			path.ConversationPostsCategory,
+			false)
+}
+
+func (bh conversationsBackupHandler) locationPath(c models.Conversationable) *path.Builder {
```
[the remark this answers is not in the document]
Comment (ryanfkeepers) [reply src/internal/m365/collection/groups/conversation_handler.go:79]: yeah, unfortunately.  there isn't really human-oriented "location" in the UX.  So we're just keeping it empty.

File: src/internal/m365/collection/groups/conversation_handler.go [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+		bh.protectedResource,
+		path.GroupsService,
+		path.ConversationPostsCategory,
+		false)
+}
+
+func (bh conversationsBackupHandler) GetItem(
+	ctx context.Context,
+	groupID string,
+	containerIDs path.Elements,
```
[the remark this answers is not in the document]
Comment (ryanfkeepers) [reply src/internal/m365/collection/groups/conversation_handler.go:95]: done

File: src/internal/operations/test/group_test.go (hunk 1 above)
Comment (ryanfkeepers) [reply src/internal/operations/test/group_test.go:192]: ah, I think that was a bad merge conflict resolution.  thanks for the catch!

**Pull request merged**